### ISSUE TITLE
Multibyte support / mbstring extension

### ISSUE THREAD

**Issue by Sarke** [stackery/php-lambda-layer#10]
Many libraries depend on this, and is usually part of a default PHP install.

It can be circumvented using the https://github.com/symfony/polyfill-mbstring library, but preferably it should be in by default.

And congrats and thanks for a nice PHP runtime lambda layer, very easy to use!

**Comment by atrope** [comment]
@Sarke you can rebuild the layer adding 
`php71-mbstring.x86_64`

in the first line of build.sh

in your php.ini you add:
`extension=mbstring.so
`

**Comment by txase** [comment]
If you can provide some evidence that it's a core extension almost always installed with a base php 7.1 build, then we could merge it into this layer.

For example, if this extension is installed by default with the core php runtime in various linux distros, then it would make sense. If, on the other hand, it's usually installed as an optional package, then it probably makes sense for people to build it into a separate layer for their specific usage.

**Comment by Sarke** [comment]
It might not be installed by default, but it's default in the sense that it is almost always installed.  Anything web-facing and relating to UTF-8 (which most things are these days) will need or want it.

I looked at a few popular ones PHP open-source projects, and Laravel, Yii, CakePHP, Drupal, WordPress, Magento, PHPUnit, all require it.

Adding to that, Symfony and CodeIgniter (and lots of others) will get by without it, but highly recommend it for performance reasons.

**Comment by txase** [comment]
@Sarke Thanks for the information. It does sound like it should be installed by default.

Would you be able to PR a change to the build.sh script as @atrope suggested above? Once you've tested that it works we can merge it in.

🙏

**Comment by jverelst** [comment]
Actually if you're trying to run a Laravel application using this layer, you run into some more extensions that need to be installed too and would make sense to add to the distro:

- iconv
- json

The first one can be easily circumvented with the 'symphony/polyfill-iconv' package, but for 'json_encode' you need the json module. That one doesn't have a polyfill available, and creating one is quite some work (I tried ...)

**Comment by txase** [comment]
@jverelst json and iconv are provided in the layer already. You simply need to specify them as extensions to load in your php.ini. Please see the README for more info.

**Comment by code-runner-2017** [comment]
Hi guys, thanks for the wonderful job. I have [forked](https://github.com/code-runner-2017/php-lambda-layer) your repo and changed it to support mbstring (as you suggested) and PDO. They are both requirements for Lumen/Laravel, that are **very** populare in the PHP community.

I also published this [post](https://medium.com/@igliop/https-medium-com-igliop-running-a-serveless-lumen-rest-api-on-aws-lambda-804089b0852c) on Medium.com on this topic.

If you are interested in bringing these small changes into the Stackery repo, I can send you a PR.

**Comment by txase** [comment]
Hi @code-runner-2017!

I would happily accept a PR for these changes :). I looked at your fork and I believe everything you changed would be reasonable to add into this layer.

Thanks!

**Comment by code-runner-2017** [comment]
Hi **@txase**! In the meanwhile I have added support for PHP 7.2 and 7.3 and submitted a PR with everything. You might decide to merge everything or just the mbstring/PDO. Details below:

1. Added support for mbstring and PDO (MySQL/Postgresql) in order to meet requirements of popular frameworks such as Lumen and Laravel;
2. Added support for PHP 7.2 and 7.3 using the REMI RPM Repository.

If you are interested just in (1), only copy changes to the build.sh file.
For (2), I have not updated the publish scripts yet.
You might want to change the README.md as well.
